arx7.js

```diff
@@ -23,7 +24,8 @@ let commands = [
   new Imgur(client),
   new Order(client),
   new Twitter(client),
-  new Youtube(client)
+  new Youtube(client),
+  new Time(client)
```

Comment:
Bit of a nit - can we insert this above Twitter? As the list of plugins grows, keeping them alphabetical would be nice